Feat: prompt.json eklendi ve prompt_service dosyası yazıldı.

Tek258 opened the pull request:
Ollama için yazılan hardcoded promptlar json haline getirilip projeye uyarlandı. Api için "prompt_service" yazıldı. olası bir hatada bana ulaşın

Tek258 committed:
Feat: prompt.json eklendi ve prompt_service dosyası yazıldı.

oguzcankaraman reviewed (changes requested):
promptları kullanılacakalrı formata getirmeli ve servisi apiden aşağı bir katmana yazmalısın

Tek258 committed:
hata

Tek258 committed:
deneme branch 'main' into feat/arda-llm
commit

Tek258 committed:
Merge branch 'main' into feat/arda-llm
the commit. önemsiz bi commit hatası

Tek258 committed:
promptlar daha düzgün hale getirildi, prompt_service.py şu anlık sadece ollama için uyumlu

Tek258 committed:
başka bölümlerinde prompt_service.py'ı kullanabilmesini kolaylaştırmak amacıyla prompts klasörü açıldı

Tek258 committed:
başka bölümlerinde prompt_service.py'ı kullanabilmesini kolaylaştırmak amacıyla prompts klasörü açıldı

Tek258 committed:
Merge branch 'feat/arda-llm' of https://github.com/oguzcankaraman/paper-mate

Tek258 committed:
fix: fazlasıyla hata düzeltmesi

Tek258 committed:
fix: page_content düzeltildi!

Tek258 committed:
git hata
Merge branch 'main' of https://github.com/oguzcankaraman/paper-mate
git hata# Lines starting with '#' will be ignored, and an empty message aborts

Tek258 committed:
Merge branch 'main' into feat/arda-llm

oguzcankaraman reviewed (approved):
lgtm
